prov/gni/include/fi_ext_gni.h

```diff
@@ -62,6 +62,7 @@ typedef enum dom_ops_val { GNI_MSG_RENDEZVOUS_THRESHOLD,
 			   GNI_MR_SOFT_REG_LIMIT,
 			   GNI_MR_HARD_REG_LIMIT,
 			   GNI_MR_HARD_STALE_REG_LIMIT,
+			   GNI_IOV_LIMIT,
```

Comment:
`GNI_IOV_LIMIT` doesn't seem to result in any memory reduction or faster code paths.  Am I missing something?  If it doesn't affect performance in anyway, then it should be removed.